src/pages/iou/request/step/IOURequestStepWaypoint.tsx

```diff
@@ -93,10 +94,8 @@ function IOURequestStepWaypoint({
             ErrorUtils.addErrorMessage(errors, `waypoint${pageIndex}`, 'bankAccount.error.address');
         }
 
-        // If the user is online, and they are trying to save a value without using the autocomplete, show an error message instructing them to use a selected address instead.
-        // That enables us to save the address with coordinates when it is selected
-        if (!isOffline && waypointValue !== '' && waypointAddress !== waypointValue) {
-            ErrorUtils.addErrorMessage(errors, `waypoint${pageIndex}`, 'distance.errors.selectSuggestedAddress');
+        if (waypointValue !== '' && waypointAddress !== waypointValue) {
+            ErrorUtils.addErrorMessage(errors, `waypoint${pageIndex}`, `distance.errors.${isOffline ? 'selectSuggestedAddress' : 'selectSuggestedAddressOrCurrentLocation'}`);
         }
```

Comment:
We still want to allow users to enter an address manually outside of the pre-defined waypoints while offline right? Or are we now preventing that to ensure that it's a valid address?

I don't remember us discussing this, maybe I forgot, but if we go this route I want to bring it up with the team more broadly. It could be a helpful solution the the fact that we don't handle it well when an invalid address is entered while offline. However, it also seems overly limiting and goes against our offline first philosophy that you should be able to do as much as possible while offline, so I think we should remove the restriction.

Reply:
> We still want to allow users to enter an address manually outside of the pre-defined waypoints while offline right?

I asked for this change since we don't allow it offline, but I wasn't aware that we allow an arbitrary address while offline. 
"It could be a helpful solution the the fact that we don't handle it well when an invalid address is entered while offline. However, it also seems overly limiting and goes against our offline first philosophy that you should be able to do as much as possible while offline, so I think we should remove the restriction." That makes total sense, I agree we should remove this restriction. 
Apologies for the confusion everyone, @pac-guerreiro, could you roll back this change please?

Reply:
@neil-marcellini @eVoloshchak 

Change is now reverted, thank you both for the help! 😄